resources/overlay/usr/local/bin/fast_page_fault_helper.c

```diff
@@ -27,6 +27,10 @@ int main(int argc, char *const argv[]) {
         perror("sigaddset");
         return 1;
     }
+    if (sigprocmask(SIG_BLOCK, &set, NULL) == -1)  {
```

Comment:
If it was not needed for page faults, do we still need it?

Reply:
Not strictly. It should still give a more consistent result, but mainly I kept it because I'm planning on reusing it for page fault latency testing